lib/instances/StarterCharacterScripts.lua

```diff
@@ -0,0 +1,4 @@
+local BaseInstance = import("./BaseInstance")
+local StarterCharacterScripts = BaseInstance:extend("StarterCharacterScripts")

```

Comment:
It's not obvious on the wiki, but `StarterCharacterScripts` actually extends `StarterPlayerScripts` for... some reason.

Here it is laid out in my local single-file wiki:
![image](https://user-images.githubusercontent.com/654598/48219633-da523480-e341-11e8-96ba-b7f853b558ec.png)

Reply:
Uh...huh. Alright.